src/Files.App/UserControls/Widgets/FileTagsWidget.xaml

```diff
@@ -47,27 +47,43 @@
 						<RowDefinition Height="Auto" />
 						<!--  Contents  -->
 						<RowDefinition Height="*" />
-						<!--  View More  -->
-						<RowDefinition Height="Auto" />
 					</Grid.RowDefinitions>
 
 					<!--  Title  -->
-					<StackPanel
-						Grid.Row="0"
-						Padding="12,8"
-						BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
-						BorderThickness="0,0,0,1"
-						Orientation="Horizontal"
-						Spacing="8">
-						<Ellipse
-							Width="12"
-							Height="12"
-							Fill="{x:Bind Color, Mode=OneWay, Converter={StaticResource StringToBrushConverter}}" />
-						<TextBlock
-							Margin="0,-2,0,0"
-							FontWeight="SemiBold"
-							Text="{x:Bind Name, Mode=OneWay}" />
-					</StackPanel>
+					<Grid BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}" BorderThickness="0,0,0,1">
+						<Grid.ColumnDefinitions>
+							<ColumnDefinition Width="Auto" />
+							<ColumnDefinition Width="*" />
+							<ColumnDefinition Width="Auto" />
+						</Grid.ColumnDefinitions>
+
+						<StackPanel
+							Padding="12,8,12,8"
+							Orientation="Horizontal"
+							Spacing="8">
+							<Ellipse
+								Width="12"
+								Height="12"
+								Fill="{x:Bind Color, Mode=OneWay, Converter={StaticResource StringToBrushConverter}}" />
+							<TextBlock
+								Margin="0,-2,0,0"
+								FontWeight="SemiBold"
+								Text="{x:Bind Name, Mode=OneWay}" />
+						</StackPanel>
+
+						<!--  View More  -->
+						<HyperlinkButton
```

Comment:
Can you add an AutomationName for the screen reader? You might have to add back the `ViewMore` string to the resource file.

Reply:
> Did you check for accessibility? You can use Accessibility Insights for this.

This step would have failed 😅

Reply:
Done